velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -3836,3 +3836,18 @@ TEST_F(DateTimeFunctionsTest, fromUnixtimeDouble) {
   });
   assertEqualVectors(expected, actual);
 }
+
+TEST_F(DateTimeFunctionsTest, toISO8601Date) {
+  const auto toISO8601 = [&](const char* dateString) {
+    return evaluateOnce<std::string, int32_t>(
+        "to_iso8601(c0)", {DATE()->toDays(dateString)}, {DATE()});
+  };
+
+  EXPECT_EQ("1970-01-01", toISO8601("1970-01-01"));
```

Comment:
Would you add some tests with negative years and years that use > 4 digits?

Reply:
Added.